source/compatibility.tex

```diff
@@ -2261,7 +2261,7 @@
 promising to return a value of a given type, and then not returning
 such a value, has always been recognized to be a questionable
 practice, tolerated only because very-old C had no distinction between
```

Comment:
Why is "very-old" hyphenated? Is there a better way to refer to pre-ANSI C?

Reply:
I don't know. That's pre-existing, though.